backend/protocol_rpc/endpoints.py

```diff
@@ -69,11 +69,30 @@ def fund_account(
 ) -> dict:
     if not accounts_manager.is_valid_address(account_address):
         raise InvalidAddressError(account_address)
-
-    transaction_id = transactions_processor.insert_transaction(
+    transaction_hash = transactions_processor.insert_transaction(
         None, account_address, None, amount, 0
     )
-    return {"transaction_id": transaction_id}
+    return {"transaction_hash": transaction_hash}
+
+
+def send_transaction(
```

Comment:
@denishacquin this method is no longer needed. Sending value is handled by the send_raw_transaction method